cmd/tailscale/main.go

```diff
@@ -233,6 +253,9 @@ func main() {
 	}
 	a.store = newStateStore(a.jvm, a.appCtx)
 	interfaces.RegisterInterfaceGetter(a.getInterfaces)
+	log.Printf("Loading apps in go: " + time.Now().String())
+	a.loadAndroidApps()
```

Comment:
I fully agree on that, this was the easiest/fastest/"hackyest" solution I could come with (with the time I spent on it), since I think calling those java methods from the UI thread is not that easy (I think there is no context there iirc), and getting the whole data stored in the class made it easier to pass the information.

I will try to explore other ways of doing it.